.changeset/ten-moose-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'wagmi': patch
```

Comment:
Keeping this as patch as it just effects type-level behavior and not runtime behavior.

From https://wagmi.sh/react/typescript

> Changes to types in this repository are considered non-breaking and are usually released as patch semver changes (otherwise every type enhancement would be a major version!).